test/common.py

```diff
@@ -359,4 +360,104 @@ def setdata(self,data):
                 self.data[i] = -1*temp;
         print(self.data)
 
-            
\ No newline at end of file
+
+class RLE:
+    """
+    These class is been created in order to store the value or the matrix which
+    are been used to test the RLE module.
+    The matrix are from github repositories for the purpose of input for
+    testing.
+    """
+    def __init__(self):
```

Comment:
Where do these values come from? What are they for?

Why is `red_pixels_1`, `self.red_pixels_1` while everything else is not assigned to self?